scte35/splicecommand.go

```diff
@@ -53,43 +54,83 @@ func parseTimeSignal(buf *bytes.Buffer) (cmd *timeSignal, err error) {
 	return cmd, nil
 }
 
+// HasPTS returns true if there is a pts time on the command.
 func (c *timeSignal) HasPTS() bool {
 	return c.hasPTS
 }
 
+// PTS returns the PTS time of the command, not including adjustment.
 func (c *timeSignal) PTS() gots.PTS {
 	return c.pts
 }
 
 type spliceNull struct {
 }
 
+// CommandType returns the signal's splice command type value.
 func (c *spliceNull) CommandType() SpliceCommandType {
 	return SpliceNull
 }
 
+// HasPTS returns true if there is a pts time on the command.
 func (c *spliceNull) HasPTS() bool {
 	return false
 }
 
+// PTS returns the PTS time of the command, not including adjustment.
 func (c *spliceNull) PTS() gots.PTS {
 	return 0
 }
 
+type component struct {
```

Comment:
It might be good to follow the convention that all type definitions remain in the `doc.go` file.